.circleci/config.yml.in

```
@@ -724,7 +724,7 @@ jobs:
   unittest_linux_cpu:
     <<: *binary_common
     docker:
-      - image: "pytorch/manylinux-cuda102"
+      - image: "pytorch/manylinux-cuda116"
```

Why do we need cuda docker images to run CPU tests? (116 is much bigger than 102, but cpu one is even smaller)
```suggestion
      - image: "pytorch/manylinux-cpu"
```